gomock/call.go

```diff
@@ -75,6 +75,9 @@ func (c *Call) Do(f interface{}) *Call {
 	return c
 }
 
+// Return sets the values to return when the target method is called.
+//
+// If Return and Do are called, the values returned via Do will be returned.
```

Comment:
As an example, this test case will fail due to panic when the call to Get is actually made:
```
func TestFunctionReturnValuePanic(t *testing.T) {
	ctlr := gomock.NewController(t)
	defer ctlr.Finish()

	mockIndex := mock_user.NewMockIndex(ctlr)

	mockIndex.EXPECT().Get("get").Do(func(interface{}) {
		return
	})

	mockIndex.Get("get")
}
```